.github/workflows/code_style_java.yml

```diff
@@ -0,0 +1,15 @@
+name: Code Style Java
```

Comment:
Please create a separate job instead in [code_style.yml](https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/code_style.yml):
```yml
jobs:
  nGraph:
  ....
  Java:
  <your code>
```